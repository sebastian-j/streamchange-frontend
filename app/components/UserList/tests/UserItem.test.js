import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import UserItem from '../userItem';

const renderer = new ShallowRenderer();

describe('<UserItem />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(
      <UserItem
        channelId="id"
        title="name"
        handleToggleUser={() => 0}
        isModerator
        isEligible
        isSponsor={false}
      />,
    );
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
