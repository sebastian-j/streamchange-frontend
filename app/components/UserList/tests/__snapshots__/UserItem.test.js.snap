// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserItem /> should render and match the snapshot 1`] = `
<li>
  <ForwardRef
    isEligible={true}
    onClick={[Function]}
    type="button"
  >
    <ForwardRef
      aria-hidden="true"
      focusable="false"
      role="presentation"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
      />
      <path
        d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
      />
    </ForwardRef>
    <ForwardRef>
      name
    </ForwardRef>
    <ForwardRef
      viewBox="0 0 16 16"
    >
      <g>
        <path
          d="M9.64589146,7.05569719 C9.83346524,6.562372 9.93617022,6.02722257 9.93617022,5.46808511 C9.93617022,3.00042984 7.93574038,1 5.46808511,1 C4.90894765,1 4.37379823,1.10270499 3.88047304,1.29027875 L6.95744681,4.36725249 L4.36725255,6.95744681 L1.29027875,3.88047305 C1.10270498,4.37379824 1,4.90894766 1,5.46808511 C1,7.93574038 3.00042984,9.93617022 5.46808511,9.93617022 C6.02722256,9.93617022 6.56237198,9.83346524 7.05569716,9.64589147 L12.4098057,15 L15,12.4098057 L9.64589146,7.05569719 Z"
        />
      </g>
    </ForwardRef>
  </ForwardRef>
</li>
`;
